packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -0,0 +1,138 @@
+ * @part footer - The footer underneath the answer.
+ */
+@Component({
+  tag: 'atomic-smart-snippet',
+  styleUrl: 'atomic-smart-snippet.pcss',
+  shadow: true,
+})
+export class AtomicSmartSnippet implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  public smartSnippet!: SmartSnippet;
+  @BindStateToController('smartSnippet')
+  @State()
+  public smartSnippetState!: SmartSnippetState;
+  public error!: Error;
+
+  /**
+   * The [heading level](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to use for the question at the top of the snippet, from 1 to 6.
+   *
+   * It's strongly recommended to set this property in order to improve accessibility.
+   */
+  @Prop({reflect: true}) public headingLevel = 0;
+
+  public initialize() {
+    this.smartSnippet = buildSmartSnippet(this.bindings.engine);
+  }
+
+  public get source() {
+    if (!this.smartSnippetState.answerFound) {
+      return null;
+    }
+    const {contentIdKey, contentIdValue} = this.smartSnippetState.documentId;
+    const linkedDocument = this.bindings.engine.state.search.results.find(
+      (result) =>
+        ResultTemplatesHelpers.getResultProperty(result, contentIdKey) ===
+        contentIdValue
+    );
+    return linkedDocument ?? null;
+  }
+
+  public readerHeading() {
+    return (
+      <Heading
+        level={this.headingLevel}
+        class="mb-4 text-xl font-bold"
+        part="heading"
+      >
+        {this.smartSnippetState.question}
+      </Heading>
+    );
+  }
+
+  public renderContent() {
+    return (
+      <atomic-smart-snippet-answer
+        htmlContent={this.smartSnippetState.answer}
+      ></atomic-smart-snippet-answer>
+    );
+  }
+
+  public renderSource() {
+    const source = this.source;
+    if (!source) {
+      return [];
+    }
+    const interactiveResult = buildInteractiveResult(this.bindings.engine, {
+      options: {result: source},
+    });
+    return (
+      <section aria-label={this.bindings.i18n.t('smart-snippet-source')}>
+        <div class="text-base mt-2 hover:underline" part="source-url">
+          <LinkWithResultAnalytics
+            title={source.clickUri}
+            href={source.clickUri}
+            interactiveResult={interactiveResult}
+            target="_self"
+          >
+            {source.clickUri}
+          </LinkWithResultAnalytics>
+        </div>
+        <div class="text-2xl mt-1 mb-2 hover:underline" part="source-title">
+          <LinkWithResultAnalytics
+            title={source.title}
+            href={source.clickUri}
+            interactiveResult={interactiveResult}
+            target="_self"
+          >
+            {source.title}
+          </LinkWithResultAnalytics>
+        </div>
+      </section>
+    );
+  }
+
+  public render() {
+    if (!this.smartSnippetState.answerFound) {
+      return <Hidden></Hidden>;
+    }
+
+    return (
+      <aside aria-label={this.bindings.i18n.t('smart-snippet')}>
```

Comment:
Is using `<aside>` , `article`, and heading the proper accessibility "recipe" ? Is this what we have for JSUI ? 🤔 

How does google do it ?

Reply:
I don't know what's the proper recipe for it, unfortunately.

I just checked Google and Google does pretty much nothing except use an extra header that contains the text "Featured snippet from the web". I'll add a similar header since I think it will legitimately improve accessibility.

Bing on the other hand seems to do pretty much nothing about accessibility.